Frontend/src/views/Account.vue

```diff
@@ -63,11 +64,24 @@ export default {
         message: "",
         user: {},
         loading: false,
+        logging: {
+          webpage: '',
+          webpageDurationStart: 0,
+        }
       }
   },
   created() {
+    // Browser logger, listener if used switches/closes tab
+    document.addEventListener('beforeunload', this.userBrowserTabSession)
+    this.logging.webpage = this.$options.name;
+    this.logging.webpageDurationStart = Date.now();
+    window.addEventListener('beforeunload', this.userBrowserTabSession)
```

Comment:
What is the difference between document.eventListener and window.eventListener?

Reply:
I think the 2 event listeners and destoryed function will make 3  separate calls to userBrowserTabSession